gradle/libs.versions.toml

```diff
@@ -31,6 +31,8 @@ jsoup = "org.jsoup:jsoup:1.17.2"
 disklrucache = "com.jakewharton:disklrucache:2.0.2"
 unifile = "com.github.tachiyomiorg:unifile:7c257e1c64"
 junrar = "com.github.junrar:junrar:7.5.5"
+common-compress = "org.apache.commons:commons-compress:1.25.0"
+xz = "org.tukaani:xz:1.9"
```

Comment:
Where is this being used 🤔

Reply:
It's in the bundles section of the same file

Reply:
Where in the codebase

Reply:
this didn't age well 😂

Reply:
mihon spyware confirmed ✅✅✅